kustomize/commands/edit/add/addreplacement.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2023 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package add
+
+import (
+	"errors"
+	"github.com/spf13/cobra"
+	"log"
+	"sigs.k8s.io/kustomize/api/types"
+	"sigs.k8s.io/kustomize/kustomize/v5/commands/internal/kustfile"
+	"sigs.k8s.io/kustomize/kyaml/filesys"
+)
+
+type addReplacementOptions struct {
+	Replacement types.ReplacementField
+}
```

Comment:
I've local tested and it didn't work fine. I suspect this field need more setup
<img width="873" alt="image" src="https://github.com/kubernetes-sigs/kustomize/assets/73322208/d31a11e7-dae5-4bc6-82cc-ea4689cc88e3">


can you take a look when you have a chance? @annasong20

Reply:
<img width="889" alt="image" src="https://github.com/kubernetes-sigs/kustomize/assets/73322208/fb134426-1b3e-408d-9b23-e95a7ded5a4e">

nvm, seems like I figure out. will tweak the code when I got more time